### ISSUE TITLE
Using my own data while find nan value in y_hat, Should I pre-scale all telemetry values between (-1,1) 

### ISSUE THREAD

**Issue by miazoin** [khundman/telemanom#3]
Trying to apply the telemanom on my own dataset, while find the nan value in the predicted "y_test" value "y_hat".
Should I pre-scale all telemetry values between (-1,1) according to the min/max in the test set?

**Comment by khundman** [comment]
You should scale all your telemetry values between (-1,1) according to the min/max in the training set (assuming you would test on a unseen future dataset in a real-world scenario). Also ensure you don't have any `NaN` values in your train/test data. See here for more discussion: https://github.com/keras-team/keras/issues/2967
